birdhouse/default.env

```diff
@@ -1,15 +1,27 @@
 # All env in this default.env can be overridden by env.local.
 # All env in this default.env must not depend on any env in env.local.
 
+# General metadata details about the server instance
+# following are the information that will be provided by Canarie-API to describe the server
+# custom values are recommended for your production server
+export SERVER_NAME="PAVICS"
+export SERVER_SYNOPSIS="The PAVICS (Power Analytics for Visualization of Climate Science) platform is a collection of climate analysis services served through Open Geospatial Consortium (OGC) protocols. These services include data access, processing and visualization. Both data and algorithms can be accessed either programmatically, through OGC-compliant clients such as QGIS or ArcGIS, or a custom web interface."
+export SERVER_INSTITUTION="Demo Server"
+export SUPPORT_URL=https://ouranosinc.github.io/pavics-sdi/support/index.html
```

Comment:
I think this one aligns with the other comment about the source. 
If it really is for the organization, it should be override-able. 
For bw compat, should be a custom override for your prod, not something for the shared repo.

Reply:
Just to be clear why I am confused.

If you think this shared repo should not have something Ouranos specific so why the new suggested value is still pavics-sdi?

If on the contrary, if having pavics-sdi here is fine, since it can be overridden, why change the value for another pavics-sdi link?  Why not just keep the same?